draft-wang-ppm-differential-privacy.md

```diff
@@ -456,9 +473,74 @@ class DpMechanism:
 
 > TODO: Specify a Gaussian sampler
 
-## Randomized Response {#rr}
+## Binary Randomized Response {#rr}
+
```

Comment:
This isn't suitable as an intro paragraph. I suggest moving it below the paragraph that mentions approximate DP and OAMC trust model.

Reply:
I opened up this section with binary randomized response's parameters and inputs right now. I mention `EPSILON_0`-DP in OC trust model now, which I think we've already defined at this point?

Reply:
Looks much better, thanks!